doc/conf.py

```diff
@@ -259,30 +259,36 @@ def source_read(_app, docname, source):
     os.chdir(file_path)
 
     filename = None
-    if docname == "arcade.color":
+    if docname == "api_docs/arcade.color":
         filename = "../arcade/color/__init__.py"
-    elif docname == "arcade.csscolor":
+    elif docname == "api_docs/arcade.csscolor":
         filename = "../arcade/csscolor/__init__.py"
 
     if filename:
+        # print(f"  XXX Handling color file: {filename}")
         import re
-        p = re.compile("^([A-Z_]+) = (\\(.*\\))")
+        p = re.compile(r"^([A-Z_]+) = (\(.*\))")
```

Comment:
What does  this regex change do?

Reply:
It replaces the string with a raw string, which does not process escapes inside it. This removes the need to escape backslashes, which in turn makes the regex inside it more legible. See [the doc for the `re` module](https://docs.python.org/3/library/re.html) for more information.